### PR TITLE
Fixtures

### PR DESCRIPTION
**Pull request by Slavka-JD**

### CODE REVIEW

**Comment by spolischook** [comment]
Not use random data fixtures. 
Use old site as a source of fixtures. 
If some data is not available on the old site - use faker.

Move all data from php to yml

File: src/AppBundle/DataFixtures/ORM/LoadData.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+namespace AppBundle\DataFixtures\ORM;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Hautelook\AliceBundle\Alice\DataFixtureLoader;
+use Nelmio\Alice\Fixtures;
+
+class LoadData extends DataFixtureLoader
+{
+    public function load(ObjectManager $om)
```
Comment (spolischook) [inline src/AppBundle/DataFixtures/ORM/LoadData.php]: why you redefine this method? 
You need only getFixtures

**Comment by spolischook** [comment]
Not use random data fixtures!

**Comment by Slavka-JD** [comment]
I don't have random data fixtures except for those they don't have on the old site

**Comment by monobook** [comment]
Orly?! 
Main rule fixtures - the should be static, if you don't have enough imagination you can use faker BUT only for generate random data for YOU. You shout to separate fixtures for test (we are talking about) and fixtures for DEMO pages.

**Pull request merged**